lib/assessment.js

```diff
@@ -8,15 +8,17 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
-
+var min = function min(p, c, i, a) {
+  var result = p;
+  if(result > c) result = c;
+  return result;
```

Comment:
Apparently, the curly braces are not _strictly_ necessary; however, I would still defer towards your including them.